resources/assets/components/CampaignIdSingle/index.js

```diff
@@ -46,7 +47,9 @@ class CampaignIdSingle extends React.Component {
   render() {
     const campaign = this.props.campaign;
     const actions = this.props.actions;
-
+    console.log(
+      format(new Date(campaign.start_date.replace(/-/g, '/')), 'MM/dd/yyyy'),
+    );
```

Comment:
Do we still want this log?

Reply:
oops we do not! Thanks for catching, @katiecrane ! Removed :)